data/categories.txt

```diff
@@ -2126,7 +2126,7 @@ fr:Arrêt de bus|2bus|arrêt|2autobus
 de:Bushaltestelle|Haltestelle|2Bus|Autobus|ÖPNV
 hu:2Buszmegálló|2busz|megálló
 id:3Halte bus
-it:3Fermata|autobus
+it:3Fermata dell'autobus|autobus
```

Comment:
Seems like you don't understand my point. "Fermata tram", or "Fermata autobus" search already should work on current app version with correct results. Please, check it out.

Your fix is good for _blue suggest_ result names on top. Instead of generic "Fermata", the app will suggest "Fermata dell'autobus" or "Fermata del tram".

Reply:
From my perspective, I'd prefer simple:

it:3autobus|3Fermata

it:3tram|3Fermata

Main transport as a first keyword for suggest. "Fermata" as a synonym for any transport ..